infra/build/functions/request_build.py

```diff
@@ -66,13 +65,12 @@ def get_empty_config():
   return build_project.Config()
 
 
-def get_build_steps(project_name, image_project, base_images_project):
+def get_build_steps(project_name):
```

Comment:
Given this change, I presume we only need `'test-project'` in [this function's corresponding test](https://github.com/google/oss-fuzz/blob/master/infra/build/functions/request_build_test.py#L50).